src/mattack_actors.cpp

```diff
@@ -184,6 +189,8 @@ bool mon_spellcasting_actor::call( monster &mon ) const
         return false;
     }
 
+    add_msg( _( spell_data.message() ), mon.disp_name(), spell_data.name() );
```

Comment:
```suggestion
    add_msg( spell_data.message(), mon.disp_name(), spell_data.name() );
```
No need to translate twice.